### PR TITLE
Do not silence stack traces

### PR DESCRIPTION
**Pull request by oharboe**
### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [ ] Did you delete any extraneous printlns/debugging code?
- [ ] Did you specify the type of improvement?
- [ ] Did you add appropriate documentation in `docs/src`?
- [ ] Did you state the API impact?
- [ ] Did you specify the code generation impact?
- [ ] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you mark the proper milestone (Bug fix: `3.4.x`, [small] API extension: `3.5.x`, API modification or big change: `3.6.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you mark as `Please Merge`?

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,7 +14,7 @@ lazy val commonSettings = Seq (
     Resolver.sonatypeRepo("releases")
   ),
   organization := "edu.berkeley.cs",
-  version := "3.5-SNAPSHOT",
```
Comment (sequencer) [inline build.sbt:17]: I think build.sbt should not be changed.

File: core/src/main/scala/chisel3/internal/Error.scala [hunk 2]
```diff
@@ -235,7 +235,6 @@ private[chisel3] class ErrorLog {
 
     if (!allErrors.isEmpty) {
       throw new ChiselException("Fatal errors during hardware elaboration. Look above for error list.")
-        with scala.util.control.NoStackTrace
```
Comment (sequencer) [inline core/src/main/scala/chisel3/internal/Error.scala:238]: Chisel have `PrintFullStackTraceAnnotation`. User can use that to print full stacks.

**Review by sequencer — CHANGES_REQUESTED** [review]
I think `PrintFullStackTraceAnnotation` can help this.
But on the other hand, I was always having issues in chisel internal tests. I think adding `PrintFullStackTraceAnnotation` to chisel internal tests sounds a good idea.

File: core/src/main/scala/chisel3/internal/Error.scala (hunk 2 above)
Comment (oharboe) [reply core/src/main/scala/chisel3/internal/Error.scala:238]: I tried PrintFullStackTraceAnnotation, didn't work.

Comment (oharboe) [reply core/src/main/scala/chisel3/internal/Error.scala:238]: I've asked some questions in https://groups.google.com/g/chisel-users/c/61vkD3EfleA

Resulting in a github issue filed [#501 ](https://github.com/ucb-bar/chiseltest/pull/501)

Comment (sequencer) [reply core/src/main/scala/chisel3/internal/Error.scala:238]: have you tried: https://github.com/ucb-bar/chiseltest/pull/501?

Comment (oharboe) [reply core/src/main/scala/chisel3/internal/Error.scala:238]: yes, doesnt work.

the error can probably be reproduced by

0.U(1.W)(0,1)

i will gove a go at creating a test case
...

Comment (oharboe) [reply core/src/main/scala/chisel3/internal/Error.scala:238]: > have you tried: [ucb-bar/chiseltest#501](https://github.com/ucb-bar/chiseltest/pull/501)?

Yes, Updated #501 with a test-case.

**Pull request closed without merging**